src/arrayMethodSort.js

```diff
@@ -1,11 +1,37 @@
 'use strict';
-
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
add here default value for the compareFunction
```suggestion
  [].__proto__.sort2 = function(compareFunction = yourFunction) {
```